spyderlib/cli_options.py

```diff
@@ -39,5 +40,8 @@ def get_options():
     parser.add_option('--profile', action='store_true', default=False,
                       help="Profile mode (internal test, "
                            "not related with Python profiling)")
+    parser.add_option('--travis', dest="travis", action='store_true',
+                      default=False,
+                      help="Run a Qtimer to stop the GUI app in travis-ci")
```

Comment:
The thing is, `travis` or `ci` are technical terms our users don't need to know or understand to the read `spyder --help`.

So if you prefer the option to remain as part of our main cli options, I think using `--test` for the option (and the help I suggested before) is just fine.